Tools/i18n/makelocalealias.py

```diff
@@ -135,8 +135,8 @@ def check(data):
     args = parser.parse_args()
 
     data = locale.locale_alias.copy()
-    data.update(parse_glibc_supported(args.glibc_supported))
     data.update(parse(args.locale_alias))
+    data.update(parse_glibc_supported(args.glibc_supported))
```

Comment:
This is separate issue (bpo-20087). Let don't mix updating to newer glibc with changing the priority of aliases tables. The latter change needs blessing from the module maintainer.